File: SpringBoot/shop-founders-modules/shop-founders-core/src/main/java/hf/shopfounders/model/GetAllShopsResponse.java
package hf.shopfounders.model;

import com.fasterxml.jackson.annotation.JsonProperty;
import hf.shopfounders.dao.DaoShop;

import java.util.List;

public class GetAllShopsResponse {

    @JsonProperty("shops")
    private List<DaoShop> shops;
    @JsonProperty("getAllShopsStatus")
    private GetAllShopsStatus getAllShopsStatus;

    public GetAllShopsResponse() {
    }

    public GetAllShopsResponse(List<DaoShop> shops, GetAllShopsStatus getAllShopsStatus) {
        this.shops = shops;
        this.getAllShopsStatus = getAllShopsStatus;
    }

    public List<DaoShop> getShops() {
        return this.shops;
    }

    public void setShops(List<DaoShop> shops) {
        this.shops = shops;
    }

    public GetAllShopsStatus getGetAllShopsStatus() {
        return this.getAllShopsStatus;
    }

    public void setGetAllShopsStatus(GetAllShopsStatus getAllShopsStatus) {
        this.getAllShopsStatus = getAllShopsStatus;
    }
}

File: AngularBoot/src/app/pages/_models/shop/getAllShopsResponse.ts
import { DaoShop } from './daoShop';
import { GetAllShopsStatus } from './getAllShopsStatus';

export class GetAllShopsResponse {

    private shops: DaoShop[];
    private getAllShopsStatus: GetAllShopsStatus;

    public getShops(): DaoShop[] {
        return this.shops;
    }

    public setShops(shops: DaoShop[]) {
        this.shops = shops;
    }

    public getGetAllShopsStatus(): GetAllShopsStatus {
        return this.getAllShopsStatus;
    }

    public setGetAllShopsStatus(getAllShopsStatus: GetAllShopsStatus) {
        this.getAllShopsStatus = getAllShopsStatus;
    }
}
